src/ray/common/ray_config_def.h

```diff
@@ -208,6 +208,9 @@ RAY_CONFIG(uint32_t, maximum_gcs_deletion_batch_size, 1000)
 /// Maximum number of items in one batch to scan from GCS storage.
 RAY_CONFIG(uint32_t, maximum_gcs_scan_batch_size, 1000)
 
+/// Maximum number of rows in GCS profile table.
+RAY_CONFIG(uint32_t, maximum_profile_table_rows_count, 1000)
```

Comment:
1000 might be too low given each of ray call might generate 4~5 profile data. When you discovered the issue, how many rows were at the profile table?